demisto_sdk/commands/content_graph/commands/update.py

```diff
@@ -54,7 +60,11 @@ def update_content_graph(
         dependencies (bool): Whether to create the dependencies.
         output_path (Path): The path to export the graph zip to.
     """
+    if is_external_repository():
+        packs_to_update = (CONTENT_PATH / "Packs").iterdir()  # type: ignore
```

Comment:
@dantavori 
Isn't packs to update supposed to be pack names?
I think in this case this is list of paths. How is it working?